src/components/MovieListPage/MovieListPage.js

```diff
@@ -22,7 +22,7 @@ export default function MovieListPage({
         />
       </CategorySelect>
       <MovieList movies={filteredMovies} borderHeight={300} />
-      <ButtonWrapper>
+      <ButtonWrapper className="margin">
```

Comment:
Why are you adding a class and not just add the styles to the `ButtonWrapper`? And if you use it again, you could use `props` instead of a class.
```suggestion
      <ButtonWrapper>
```